Tasks/WindowsMachineFileCopyV1/Utility.ps1

```diff
@@ -89,6 +89,15 @@ function Validate-DestinationPath(
         ThrowError -errorMessage (Get-LocalizedString -Key "Remote destination path '{0}' cannot contain environment variables." -ArgumentList $value)
     }
 }
+
+function Validate-AdditionalArguments([string]$additionalArguments)
+{    
+    if($additionalArguments -match "[&;]")
+    {
+        ThrowError -errorMessage (Get-LocalizedString -Key "Additional arguments can't include separator characters '&' and ';'. Please verify input. To learn more about argument validation, please check https://aka.ms/azdo-task-argument-validation")
```

Comment:
As we discussed, we can move it localization file instead of using hard-coded string.

Reply:
I talked to Maxim Petrov from Akvelon about this task and, unfortunately, the first version doesn't support messages translation through the localization resources. That's why all the messages are hardcoded.